tests/helper/helper_oc.go

```diff
@@ -126,7 +126,7 @@ func (oc *OcRunner) CheckCmdOpInRemoteCmpPod(cmpName string, appName string, prj
 	podName := strings.Replace(outPodName, "'", "", -1)
 	session := CmdRunner(oc.path, append([]string{"exec", podName, "--namespace", prjName,
 		"-c", cmpDCName, "--"}, cmd...)...)
-	Eventually(session).Should(gexec.Exit(0))
+	Eventually(session).Should(gexec.Exit())
```

Comment:
This is an important change because when we run a failing command like `listing a file that doesn't exist` the session exits with a non-zero exit code. This check wouldn't help us verifying such failing commands and hence we need to remove it.

Reply:
As we are not validating the exit returned code then I don't think line `Eventually(session).Should(gexec.Exit())` is required over here.

`stdOut` and `stdErr` will not be affected anyway.

Reply:
actually if you read the docs, the checking of `returned code` is optional. This statement confirms that the session exits successfully. so its still necessary.

Reply:
@girishramnani Right, but when this return code is not being validated and in the very next line we are waiting for the `stderr` and `stdout` which itself confirms that session has completed. So, I think we don't need the statement. I might be wrong but it would be great if you check locally after removing whatever I am suggesting.

Reply:
Removed